src/mpi/errhan/errnames.txt

```diff
@@ -938,6 +938,11 @@ is too big (> MPIU_SHMW_GHND_SZ)
 **mpl_gavl_create: mpl_gavl_create failed
 **mpl_gavl_delete_range: mpl_gavl_delete_range failed
 **mpl_gavl_delete_start_addr: mpl_gavl_delete_start_addr failed
+
+## Async thread affinity related error messages
+**parse_thread_affinity:Failed to parse thread affinity
+**set_thread_affinity:Failed to set thread affinity
```

Comment:
Can we add a long message (with the parsed string and the affinity mask, etc.)?

Reply:
Added long message for `parse_thread_affinity`